New log-publish-time configuration option (#37)
eProsima/DDS-Recorder: docs/rst/recording/usage/configuration.rst

In other words, the ``event-window`` acts as a sliding time window that allows to save the collected samples in this time window only when the remote controller event is received.
By default, its value is set to ``20`` seconds.

> Log PublishTime

Reviewer: ```suggestion
Log Publish Time
```

^^^^^^^^^^^^^^^

> By default (``log-publishTime: false``), received messages are stored in the MCAP container with a ``logTime`` equals the reception timestamp.

Reviewer: ```suggestion
By default (``log-publishTime: false``) received messages are stored in the MCAP file with ``logTime`` value equals to the reception timestamp.
```
